js/components/skipControls.js

```diff
@@ -276,7 +314,9 @@ class SkipControls extends React.Component {
 
     const className = classNames('oo-skip-controls', {
       'oo-inactive': this.props.isInactive,
-      'oo-in-background': this.props.isInBackground
+      'oo-in-background': this.props.isInBackground,
+      'oo-absolute-centered': true,
```

Comment:
Since these are related to layout, could we expose a `className` prop so that parent components can set the class that works with their layout?

Reply:
Will add